build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -7,9 +7,9 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class DependencyVersions
     {
-        public static readonly string CoreCLRVersion = "1.0.2";
-        public static readonly string JitVersion = "1.0.2";
-        public static readonly string CoreCLRVersion_Latest = "1.1.0-beta-24415-01";
-        public static readonly string JitVersion_Latest = "1.1.0-beta-24415-01";
+        public static readonly string CoreCLRVersion = "1.1.0-preview1-24508-06";
+        public static readonly string JitVersion = "1.1.0-preview1-24508-06";
+        public static readonly string CoreCLRVersion_Latest = "1.1.0-preview1-24508-06";
+        public static readonly string JitVersion_Latest = "1.1.0-preview1-24508-06";
```

Comment:
Given that we want master of Core-Setup to also use non-stable versions of the JIT and CoreCLR (e.g. daily builds that are published from pipeline), it makes sense to have a single version.

@ianhays Can you please make this fix in master and release/1.1.0 branches (in the latter once I have merged this PR) after validating that Arm64 is not broken?